nep-22.mediawiki

```diff
@@ -0,0 +1,67 @@
+<pre>
+  NEP: 22
+  Title: Contract Update Function
+  Author: Owen Zhang <zhangzhihong@ngd.neo.org>, Fernando Díaz Toledano <shargon@gmail.com>, Erik Zhang <erik@neo.org>, Roman Khimov <roman@nspcc.ru>
+  Type: Standard
+  Status: Accepted
+  Created: 2021-04-19
+</pre>
+
+
+==Abstract==
+
+This proposal outlines a standard for the NEO blockchain that will provide systems with a generalized interaction mechanism for smart contract update.
+
+==Motivation==
+
+As the NEO blockchain scales, Smart Contract Update will become increasingly important. Update mechanism is already there from initial N3 release. But we haven't summarized that standard. That's why this standard is necessary.
```

Comment:
It's more like we didn't have an agreement on how deployed contracts expose this functionality to their admins. Because ContractManagement itself doesn't care about code that calls it.